### ISSUE TITLE
Eror getting config

### ISSUE THREAD

**Issue by ysomad** [xssnick/tonutils-go#285]
Hi! I'm getting error on parsing mainnet config, issue encountered between Dec. 10 and 11.

Code:
```go
cfg, err := liteclient.GetConfigFromUrl(ctx, "https://ton.org/global.config.json")
if err != nil {
return nil, fmt.Errorf("get config: %w", err)
}
```

Returns:
`get config: invalid character '<' looking for beginning of value`

**Comment by iw4p** [comment]
Hi, Yes I can confirm it!

**Comment by iw4p** [comment]
> It's caused because in response HTML is present, GetConfigFromUrl not handling status of response

any quick fix for it?

**Comment by ysomad** [comment]
idk, vpn or proxy to bypass cloudflare

**Comment by ysomad** [comment]
> `curl https://ton-blockchain.github.io/global.config.json` and `curl https://ton.org/global.config.json` works fine on both VPS and my machine! Using VPN or proxy for what when I can get the result using curl but it does not work with tonutils-go?

Yeah, you're right. I'm pretty sure the problem may caused by missing headers, User-Agent maybe, will check later

**Comment by xssnick** [comment]
Same for me, looks like some rules on the server blocks it, i will notify the team. 

As an option for now you can download it and put locally, then load with `liteclient.GetConfigFromFile("path to file.json")`

**Comment by iw4p** [comment]
I tested both absolute and relative path 
open /root/xyz/mainnet-global.config.json: no such file or directory"
open ../mainnet-global.config.json: no such file or directory"

**Comment by iw4p** [comment]
does it work for you? @ysomad I still get the same error.
Any update when it can fixed? @xssnick

**Comment by xssnick** [comment]
> does it work for you? @ysomad I still get the same error. Any update when it can fixed? @xssnick

You getting error even when using file? URL `https://ton-blockchain.github.io/global.config.json` works fine for me, for `ton.org` I already notified them, hope will be fixed soon

**Comment by iw4p** [comment]
> > does it work for you? @ysomad I still get the same error. Any update when it can fixed? @xssnick
> 
> You getting error even when using file? URL `https://ton-blockchain.github.io/global.config.json` works fine for me, for `ton.org` I already notified them, hope will be fixed soon

Thanks! I fixed it, problem was on my side! But it'd be good if it fixes soon from the URL!

One more question, how often the .json will change? can I keep it as a file for how long?

**Comment by khuongdo95** [comment]
https://github.com/xssnick/tonutils-go/pull/289
@xssnick i have recomend fix this, i tested ok, if ok merge it
